src/content/theme.md

```diff
@@ -1,7 +1,7 @@
 ## 10 years of pushing the boundaries
 
-### Work in progress
-
-This year we want to shift our attention to the decade of progress in the world of frontend development. Back in 2013, we went through a period of thinking and rethinking best practices and frontend development took off as one of the most common disciplines in tech. Frameworks like React and Vue now enter into their second decade of existence.
+This year we want to shift our attention to the decade of progress in the world of JavaScript. Back in 2013, we went through a period of rethinking practices as frontend development took off and became one of the most common disciplines in tech. Framework ecosystems like React, Angular or Vue now enter into their second decade of existence, while web capabilities and standards are increasingly more potent and versatile.
```

Comment:
```suggestion
This year we want to shift our attention to the decade of progress in the world of web development. Back in 2013, we went through a period of rethinking practices as frontend development took off and became one of the most crucial disciplines in tech. Framework like React, Angular or Vue and their ecosystems now enter into their second decade of existence, while web capabilities and standards are increasingly more potent and versatile.
```

Reply:
Love the `crucial` vs `common`, great idea!